src/test/java/core/basesyntax/fruitoperation/strategy/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.fruitoperation.strategy;
+
+import core.basesyntax.fruitoperation.Operation;
+import core.basesyntax.fruitoperation.OperationBalance;
+import core.basesyntax.fruitoperation.OperationPurchase;
+import core.basesyntax.fruitoperation.OperationReturn;
+import core.basesyntax.fruitoperation.OperationSupply;
+import core.basesyntax.fruitoperation.Operations;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.FruitServiceImpl;
+import core.basesyntax.service.file.DataReader;
+import core.basesyntax.service.file.impl.FileReader;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class OperationStrategyImplTest {
+    private static FruitService fruitService;
```

Comment:
again, let's check the work of each Operation separately, as well as OperationStrategy, if you want

do not use fruitservice or reader, communicate with storage directly and clear it after each test

Reply:
done